app/schemas/article.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+from typing import Any, Dict, Generic, List, Optional, TypeVar
+
+from pydantic import BaseModel, Field
+
+# T는 공변성을 갖는 타입 변수
+T = TypeVar("T", covariant=True)
+
+
+class ArticleDTO(BaseModel, Generic[T]):
+    """
+    뉴스 기사 DTO 모델
+
+    공변성을 만족하는 제네릭 설계로, 확장 가능한 뉴스 기사 데이터 구조 제공
+    """
+
+    title: str
+    link: str
```

Comment:
이름이 직관적이지 않다는 생각이 들어서 개인적으론 필드 이름이 조금 어색해보입니다

```suggestion
    url: str
```

둘 중 괜찮다 생각하는 필드명을 선택해보시죠~

Reply:
url 로 수정하겠습니다.